src/utils/split_text_by_new_line.tsx

```diff
@@ -6,7 +6,7 @@ import * as React from 'react';
  */
 function splitTextByNewLine(text: string): JSX.Element[] {
     if (text) {
-        return text.split("\n").map((p, i) => <p key={i}>{p}</p>);
+        return text.split("\n").map((p, i) => <p key={i}>{p || "&nbsp;"}</p>);
```

Comment:
This just prints out `&nbsp;` rather than adding a space. Also, we should remove the margin-bottom from the p for the textarea component